src/cozy/wrappers/sharing/sharing.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { AngularWrapperComponent, AngularWrapperProps } from '../angular-wrapper.component';
+// @ts-ignore
+import Sharing from './sharing.jsx';
+
+interface SharingProps extends AngularWrapperProps {
+    file: any;
+}
+
+@Component({
+    selector: 'app-sharing',
+    templateUrl: '../angular-wrapper.component.html',
+    encapsulation: ViewEncapsulation.None,
+})
+export class SharingComponent extends AngularWrapperComponent {
+    @Input() collectionId: string = null;
+
+    /******************/
+    /* Props Bindings */
+    /******************/
+
+    protected async getProps(): Promise<SharingProps> {
+        const collections = await this.collectionService.getAllDecrypted();
+
+        const currencCollection = collections.find(collection => collection.id === this.collectionId);
+
+        const data = {
+            extension_installed: true,

```

Comment:
It was kind of wanted, `extension_installed` is a requirement for `ReactWrapper` component but is used only in the `installation-page` context, so on other components I inject `true` instead of querying the stack.

But you pointed out that we need to declare it on all components with this fake value. So it has been moved to parent component in this PR https://github.com/cozy/cozy-pass-web/pull/34